### DISCUSSION TITLE
Slack build notification is stuck at deploying when the build step is ignored

### DISCUSSION THREAD

**Discussion by MichelML** [vercel/vercel#6453] [Ideas]
When the build gets cancelled with https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel, the slack notification stays at "deploying". See images below

![image](https://user-images.githubusercontent.com/9539763/124975310-e5caf180-dffb-11eb-9e26-5fa5abc7f152.png)

![image](https://user-images.githubusercontent.com/9539763/124975434-06934700-dffc-11eb-91a6-90a2c031e45d.png)

It would be nice to update the Slack notification to red or grey, and replace the "Deploying" text to "Cancelled" or "Ignored"

**Comment by MichelML** [comment]
@vercel-infra any thoughts on this?

**Comment by leerob** [comment]
Thanks for the feedback! I will relay this to the Integrations team.
